packages/core/content-manager/admin/src/pages/EditView/utils/data.ts

```diff
@@ -10,6 +10,7 @@ import type { Schema, UID } from '@strapi/types';
  * traverseData
  * -----------------------------------------------------------------------------------------------*/
 
+type PartialSchema = Partial<Schema.Schema> & Pick<Schema.Schema, 'attributes'>;
```

Comment:
Worth a comment to explain why we want to support a Partial Schema? I assume because you don't store the whole schema in CH? 🤔

Reply:
Yes CH only saves attributes, and only attributes are used in those functions